extensions/uuid-generator/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "uuid-generator",
+  "title": "UUID Generator",
+  "description": "A quick way to generate UUIDs without opening the browser",
+  "icon": "command-icon.png",
+  "author": "jmaeso",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "generate",
+      "title": "Generate UUID to clipboard",
+      "description": "Copy to clipboard a randomly generated UUID",
```

Comment:
```suggestion
      "description": "Copy a randomly generated UUID to clipboard",
```

Reply:
I think this was already addressed by @fe9lix but you commented into the original commit. Will discard. Suggestion into the right direction though.